src/relay/op/tensor/reduce.cc

```diff
@@ -353,6 +353,43 @@ Example::
 .set_attr<TOpPattern>("TOpPattern", kCommReduce);
 
 
+Array<Tensor> AllCompute(const Attrs& attrs,
+                         const Array<Tensor>& inputs,
+                         const Type& out_type,
+                         const Target& target) {
+  return ReduceCompute(attrs, inputs, out_type, target, topi::all);
+}
+
+
+RELAY_REGISTER_REDUCE_OP("all")
+.describe(R"code(Computes the logical AND of array elements over given axes.
+
+Example::
+
+  data = [[[ True,  True,  True],
```

Comment:
Does this operator allow numerical type? If so. we'd better update the doc.

Reply:
just allow boolean tensor, have updated the doc